.env.development

```diff
@@ -1,3 +1,4 @@
+REACT_APP_PLAY_API_URL='https://api.github.com/repos/reactplay'
```

Comment:
The value is by default a string in the environment file. You no need to add a quote here. So please change it. Also, you need to update the ReactPlay doc to include this new entry.

https://docs.reactplay.io/local-development#create-a-env-file-at-the-root-of-your-project-folder-with-the-following-content

Otherwise, it looks great, We have tested it. Once you make the changes, we shall merge it. @sudipkundu999

Reply:
I've removed the quotes in .env
For docs, I'll raise a PR in the docs repo

Reply:
@sudipkundu999 Cool, share the PR link here once done.

Reply:
- in `.env.development` there are these many variables
```
REACT_APP_PLAY_API_URL=https://api.github.com/repos/reactplay
REACT_APP_NHOST_BACKEND_URL=https://rgkjmwftqtbpayoyolwh.nhost.run
REACT_APP_NHOST_VERSION=v1
REACT_APP_NHOST_ENDPOINT=graphql
REACT_APP_PLAY_WEB_SVC=https://api.reactplay.io/.netlify/functions/server
DISABLE_ESLINT_PLUGIN=true
REACT_APP_ACTIVITIES_ON=true
REACT_APP_ACTIVITY_ID=hackrplay
REACT_APP_DADJOKES_URL=https://jokeapi-v2.p.rapidapi.com/joke/
REACT_APP_DADJOKES_APIKEY='b71df95c75msha446fab91d0e935p1d0262jsn1d938cb85502'
REACT_APP_DADJOKES_APIHOST='jokeapi-v2.p.rapidapi.com'
```

- in the docs, it only has 
```
REACT_APP_NHOST_BACKEND_URL=https://rgkjmwftqtbpayoyolwh.nhost.run/
REACT_APP_NHOST_VERSION=v1
REACT_APP_NHOST_ENDPOINT=graphql
```

@atapas should I add all the variables not present in the doc or only add my new env?

Reply:
@sudipkundu999 None of the others are needed for people to start contributing.. so adding this one alone is good.

Reply:
https://github.com/reactplay/docs/pull/75/files